src/app/courses/breadcrumbs/breadcrumbs.component.html

```diff
@@ -1,5 +1,9 @@
 <nav aria-label="breadcrumb">
   <ol class="breadcrumb">
-    <li class="breadcrumb-item active">Courses</li>
+    <li class="breadcrumb-item active"
+        *ngFor="let path of paths">
+      <a *ngIf="path.link" routerLink="{{path.link}}">{{path.title}}</a>
```

Comment:
Also you may eliminate interpolation with `[routerLink]="path.link"`
(when you use attribute without square brackets, your param is a string, but when you use brackets - it's an object)